rfcs/text/0000-multi-data-collection-dataset-processing.md

```diff
@@ -0,0 +1,233 @@
+### DCP PR:
+
+***Leave this blank until the RFC is approved** then the **Author(s)** must create a link between the assigned RFC number and this pull request in the format:*
+
+`[dcp-community/rfc#](https://github.com/HumanCellAtlas/dcp-community/pull/<PR#>)`
+
+# Processing Datasets that Span Multiple Data Collection Runs
+
+## Summary
+This RFC proposes a design solution to allow datasets that span multiple data collection runs to be processed in unison as per the scientific requirements. The proposal generalizes to multiple data type modalities, but also provides specific details for 10X V2 3’ single-cell data as an example and to address an immediate need for a data processing solution.
+
+
+## Author(s)
+[Nick Barkas](mailto:barkasn@broadinstitute.org),
+[Mark Diekhans](mailto:markd@ucsc.edu)
+
+## Shepherd
+-[Nick Barkas](mailto:barkasn@broadinstitute.org)
+
+## Motivation
+Correct data processing requires the initiation of data processing pipelines with all the appropriate input data. The current mechanism of bundle notifications is not effective when data spans multiple bundles because there is no clear set of rules to identify the bundles that need to be co-processed.  Limiting the downstream process to the scope of a single bundle requires the submitter to define the scope of the processing without any foreknowledge of future processes needs.
+
+An immediate need to process 10X V2 scRNA-seq datasets that span multiple bundles exists. The need for co-processing multiple bundles is not however limited to this scenario and extends to any other NGS modality that involves repeated sequencing of the same library. Furthermore the need to co-process data may extend to other data modalities that the project will accept in the future.
```

Comment:
Can you give a clear example of the immediate need? if this is the library sequencing over multiple lanes/plexes of sequencing there has been an adhoc solution in this instance and the library prep entity provides another solution. In that context, there is no current need for multi-bundle processing. If this is another example, please provide it

Reply:
I am confused.  So is data being bundled by library prep? 

If there is not an immediate needed, that would lower the priority of implementation, how this would be good to have a design in place.  Rebundling primary submissions based on the needs of analysis pipelines is problematic.

Reply:
The ICA and pe'er dataset were fixed in an adhoc manner and the RFC proposed by @malloryfreeberg would enable bundling on library prep rather than cell suspension which would provide resolution 

https://github.com/HumanCellAtlas/dcp-community/pull/87